examples/regularized_mnist.py

```diff
@@ -1,3 +1,11 @@
+
+"""
+Regularized MNist Example
+================================
+
+TODO
```

Comment:
@bstriner could you maybe write a few things about this example.
All exampes in the examples folder must have this little docstring in the format above:
Theses examples are rendered as shown here https://inferno-new.readthedocs.io/en/latest/auto_examples/regularized_mnist.html#sphx-glr-auto-examples-regularized-mnist-py

Reply:
@DerThorsten Yup! Take a look here: https://github.com/inferno-pytorch/inferno/pull/121